### PR TITLE
response classes updated with current json responses

### PR DESCRIPTION
**Pull request by rodkings**
Repository.cs basic re-factor.

### CODE REVIEW

File: src/Untappd.Net.UnitTests/Untappd.Net.UnitTests.csproj.bak [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    <Reference Include="System.Core" />
+    <Reference Include="System.Xml.Linq" />
+    <Reference Include="System.Data.DataSetExtensions" />
+    <Reference Include="Microsoft.CSharp" />
+    <Reference Include="System.Data" />
+    <Reference Include="System.Xml" />
+  </ItemGroup>
+  <ItemGroup>
+<<<<<<< HEAD
+    <Compile Include="Responses\TestDeserializer.cs" />
```
Comment (TerribleDev) [inline src/Untappd.Net.UnitTests/Untappd.Net.UnitTests.csproj.bak]: this file looks like it has merge conflicts inside of it. Do we need this file?

File: src/Untappd.Net/Untappd.Net.csproj.bak [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    <Reference Include="System.Xml.Linq" />
+    <Reference Include="System.Data.DataSetExtensions" />
+    <Reference Include="Microsoft.CSharp" />
+    <Reference Include="System.Data" />
+    <Reference Include="System.Xml" />
+  </ItemGroup>
+  <ItemGroup>
+<<<<<<< HEAD
+    <Compile Include="SingleObjectArrayConverter.cs" />
+=======
```
Comment (TerribleDev) [inline src/Untappd.Net/Untappd.Net.csproj.bak]: do we need this file?

File: src/Untappd.Net.UnitTests/Responses/Json/BeerSearch.json [hunk 3]
```diff
@@ -0,0 +1,696 @@
+﻿{
+    "meta": {
+        "code": 200,
+        "response_time": {
+            "time": 0.156,
+            "measure": "seconds"
+        },
```
Comment (TerribleDev) [inline src/Untappd.Net.UnitTests/Responses/Json/BeerSearch.json:7]: Do we actually need these .json files?

Comment (rodkings) [inline src/Untappd.Net.UnitTests/Responses/Json/BeerSearch.json:7]: They are for json deserialize unit testing purposes.

**Pull request merged**